tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py

```diff
@@ -8,154 +8,238 @@
+                        result_field='RACE_BLACK', result_function=validators.oneOf((1, 2)),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='RACE_HAWAIIAN', result_function=validators.oneOf((1, 2)),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='RACE_WHITE', result_function=validators.oneOf((1, 2)),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='MARITAL_STATUS', result_function=validators.isInLimits(1, 5),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='PARENT_WITH_MINOR_CHILD', result_function=validators.isInLimits(1, 3),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='EDUCATION_LEVEL', result_function=validators.or_validators(
+                            validators.isInStringRange(0, 16),
+                            validators.isInStringRange(98, 99)
+                        ),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                        result_field='CITIZENSHIP_STATUS', result_function=validators.oneOf((1, 2)),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='COOPERATION_CHILD_SUPPORT', result_function=validators.oneOf((1, 2, 9)),
+                    ),
+                validators.validate__FAM_AFF__HOH__Fed_Time(),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='EMPLOYMENT_STATUS', result_function=validators.isInLimits(1, 3),
+                    ),
+                validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                        result_field='WORK_ELIGIBLE_INDICATOR', result_function=validators.or_validators(
+                            validators.isInStringRange(1, 9),
+                            validators.matches('12')
```

Comment:
OFA added value 11 on 7/27/2023. Please include it here as well. workbook updated.